packtools/sps/validation/article_and_subarticles.py

```diff
@@ -244,23 +281,35 @@ def validate_article_type(self, article_type_list=None):
         article_type_list = article_type_list or self.article_type_list
 
         if not article_type_list:
-            raise ValidationArticleAndSubArticlesArticleTypeException("Function requires list of article types")
+            raise ValidationArticleAndSubArticlesArticleTypeException(
+                "Function requires list of article types"
+            )
 
         article_type_list = [tp.lower() for tp in article_type_list]
 
         validated = article_type in article_type_list
-        yield {
-            'title': 'Article type validation',
-            'xpath': './article/@article-type',
-            'validation_type': 'value in list',
-            'response': 'OK' if validated else 'ERROR',
-            'expected_value': article_type_list,
-            'got_value': article_type,
-            'message': 'Got {} expected one item of this list: {}'.format(article_type, " | "
-                                                                          .join(article_type_list)),
-            'advice': None if validated else 'XML has {} as article-type, expected one item of this list: {}'
-            .format(article_type, " | ".join(article_type_list))
-        }
+
+        data = self.articles.data[0]
+        data.update({
+            "specific_use": self.articles.main_specific_use,
+            "dtd_version": self.articles.main_dtd_version
```

Comment:
@Rossi-Luciano ausência deles é erro do tipo CRITICAL. Acho que na dúvida, adiciona o parâmetro de error_level e deixa como default o valor CRITICAL.

Então, para as validações do tipo value, value in list, tudo é CRITICAL

```python
def function(self, error_level=None):
      error_level = error_level or "CRITICAL"
```